src/pages/market/components/AllItemsSection.jsx

```diff
@@ -8,16 +8,7 @@ import PaginationBar from '../ui/PaginationBar';
 
 const getPageSize = () => {
   const width = window.innerWidth;
-  if (width < 768) {
-    // Mobile viewport
-    return 4;
-  } else if (width < 1280) {
-    // Tablet viewport
-    return 6;
-  } else {
-    // Desktop viewport
-    return 10;
-  }
+  return width < 768 ? 4 : width < 1280 ? 6 : 10;
```

Comment:
```suggestion
  return width < 768 ? 4 : width < 1280 ? 6 : 10;
```
* 768,4,1280,6,10 같은 정수는 의미가 불분명해서 변수로 할당해 의미를 명확하게 하는게 좋습니다.
예)
`const TABLET_WIDTH = 768`
`const TABLE_PAGE_SIZE = 4`
...

* 중첩된 삼항연산자는 읽기 어려울 수도 있습니다. 
* 삼항연산자는 간단한 식일때 사용하는게 좋고, 중첩된 경우 조건문으로 풀어서 쓰는게 좋습니다.